package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@aragon/osx-commons-repo",
+  "license": "AGPL-3.0-or-later",
+  "private": true,
+  "workspaces": {
+    "packages": [
+      "contracts",
+      "subgraph"
+    ],
+    "nohoist": [
+      "**/@graphprotocol/graph-ts",
+      "**/@graphprotocol/graph-ts/**",
+      "**/@graphprotocol/graph-cli",
+      "**/@graphprotocol/graph-cli/**",
+      "**/matchstick-as",
+      "**/matchstick-as/**"
+    ]
```

Comment:
I copied this one from the hardhat template repo. Should we be careful about hoisting (https://classic.yarnpkg.com/blog/2018/02/15/nohoist/) @mathewmeconry